naxsi_core.rules

```diff
@@ -0,0 +1,94 @@
+# SecRulesEnabled;
```

Comment:
i do not see any reason to have this `naxsi_core.rules` file here, since it is already into the main repository.

Reply:
Agreed, especially since it seems to be less restrictive than the original one (lot rules doesn't check cookie here, even some of them are missing)